artview/components/select_region.py

```diff
@@ -155,16 +151,19 @@ def button_press_callback(self, event):
         Grab the data when line is drawn.
         If shape is closed, then compile points within.
         '''
+        # test for double click support (matplotlib > 1.2)
+        db_support = 'dblclick' in dir(event)
         if event.inaxes:
```

Comment:
this is dependent of matplotlib version, not OS. If matplotlib is correctly interpreting the OS, is an other question, but the right click close is still working.